metrics/README.md

```diff
@@ -25,3 +27,33 @@ the name of the metric. The second one contains the value (float or integer).
 The third one contains the description of the metric (in general, NOT
 for this particular file). A space is mandatory between the first and the
 second column, and between the second and the third columns.
+
+## Metrics in make
+
+Metrics could be calculated in **measure** step presented in makefile.
+Firstly, executed `steps/measure.sh` that performs preparation steps:
+
+* Creating necessary files and directories
+* Collecting jobs to execute in parallel
+  (Currently, one for each java file to analyze metrics for)
+* Executes jobs using `help/parallel.sh`
```

Comment:
@IlnurHA why it's creating, collecting and then all of a sudden executes? Set the time of the verbs straight